Libraries/Utilities/MatrixMath.js

```diff
@@ -581,7 +581,7 @@ var MatrixMath = {
       skew,
       translation,
 
-      rotate: rotationDegrees[2],
+      perspectiveZ: perspective[2] != 0 ? -1/perspective[2] : 1280,
```

Comment:
One thing that I couldn't manage to do is to retrieve the pixel density, as in Android View docs:
```java
float scale = context.getResources().getDisplayMetrics().density;
view.setCameraDistance(distance * scale);
```

> The default distance depends on the screen density. For instance, on a medium density display, the default 
> distance is 1280. On a high density display, the default distance is 1920."